src/main/kotlin/me/serce/solidity/lang/resolve/engine.kt

```diff
@@ -28,13 +28,17 @@ object SolResolver {
           resolveEvent(element) +
           resolveContract(element) +
           resolveEnum(element) +
-          resolveUserDefinedValueType(element)
+          resolveUserDefinedValueType(element) +
+          resolveConstant(element)
       } else {
-        resolveContract(element) +
+        resolveError(element) +
+          resolveContract(element) +
+          resolveEvent(element) +
           resolveEnum(element) +
           resolveStruct(element) +
           resolveUserDefinedValueType(element) +
-          resolveAliases(element)
+          resolveAliases(element) +
+          resolveConstant(element)
       }
```

Comment:
both branches are almost identical now. 
please extract the common part out of the `if` statement to simplify the code